src/pluto/pluto.py

```diff
@@ -278,10 +278,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
                 fig = generate_graph(
                     v1v2, v2v3, v1v2_total, v2v3_total, group["columns"]
                 )
-                fig.update_layout(title=group["title"], template="plotly_white")
+                fig.update_layout(
```

Comment:
I know I suggested this code, but the issue with these change is that the titles for the mismatch graphs are a different than the titles for aggregate, nulls, etc. Making the titles consistent would require a more complex implementation which I why I suggested we skip. Happy to help implement a DRY way to make the titles bigger if it's interesting to you Andrew

Reply:
fair enough, let's just save it for seperate issue to do them all. We probably want to combine it with in general making universal styling defaults - the plotly_white thing probably doesn't need to be everywhere either

Reply:
added #127